app/assets/javascripts/admin/datastore_health_check.js

```diff
@@ -26,11 +26,38 @@ export const pingDataStoreIfAppropriate = memoizedThrottle(async (requestedUrl:
   if (usedDataStore != null) {
     const { url } = usedDataStore;
     const healthEndpoint = `${url}/data/health`;
-    Request.triggerRequest(healthEndpoint, { doNotCatch: true, mode: "cors", timeout: 5000 }).catch(
-      () => {
+    Request.triggerRequest(healthEndpoint, { doNotInvestigate: true, mode: "cors", timeout: 5000 })
+      .then(async () => {
+        const buildinfoWebknossos = await RestAPI.getBuildInfo();
+        const webknossosVersion = buildinfoWebknossos.webknossos.gitTag;
+        const webknossosOldestSupportedVersion =
+          buildinfoWebknossos.webknossos.oldestSupportedVersion;
+
+        const buildinfoDataStore = await RestAPI.getDataStoreBuildInfo(url);
+        if (buildinfoDataStore.webknossosDatastore) {
```

Comment:
Is this an appropriate way to handle this in dev-mode (without standalone datastore)? The problem is that both `getBuildInfo()` and `getDataStoreBuildinfo()` redirect to `localhost:9000/api/buildinfo` but it only shows the buildinfo of webknossos and not the datastore